source/reference/about-mongosync.txt

```diff
@@ -0,0 +1,131 @@
+.. _about-mongosync:
+
+========================
+About ``mongosync``
+========================
```

Comment:
The heading markers should align with the length of the title (sometimes spacing comes out a little funky on GitHub so as long as it is aligned on vscode, its all set)

```suggestion
===================
About ``mongosync``
===================
```

Reply:
Fixing now!